wavefront/wavefront.go

```diff
@@ -191,7 +247,8 @@ func writeMetrics(cfg *config.Params, insts []*metric.InstanceMsg) {
 				log.Warnf("couldn't translate metric value: %s %v, err: %v", metricName, value, err)
 			} else {
 				deltaMetricName := wf.DeltaCounterName(metricName)
-				counter := wf.GetOrRegisterMetric(deltaMetricName, metrics.NewCounter(), tags).(metrics.Counter)
+				counter := metrics.NewCounter()
```

Comment:
same as above, Should we use https://github.com/wavefrontHQ/go-metrics-wavefront/blob/3fca929deab620bfd77b9749a5f0cf221c10e62b/reporting/reporter.go#L364 here?

Reply:
changes done as per comment.